### ISSUE TITLE
New Maintainer for the library? 

### ISSUE THREAD

**Issue by alaminopu** [xgfe/react-native-datepicker#378]
First of all, Thank you guys for building this library. So many people are using it. New issues are coming and some pull requests are there too. It seems like @feyy is not getting enough time to manage this library anymore. So I think it's very important that someone should take over managing the library. 

Can we discuss more and find a new maintainer from the community? @feyy

**Comment by karltaylor** [comment]
Could it be possible to hand over to https://github.com/react-native-community?

**Comment by karltaylor** [comment]
Turns out they already have https://github.com/react-native-community/react-native-datetimepicker

**Comment by rdewolff** [comment]
Yeah am wondering what's the futur of this repo VS the official `rn-datetimepicker` repo. What do you think people?
